modules/common-attributes.adoc

```diff
@@ -42,5 +42,12 @@ endif::[]
 :ocp: OpenShift Container Platform
 :rhacs-version: 3.0.62.0
 :ocp-supported-version: 4.5
+<<<<<<< HEAD
```

Comment:
remove this line

Reply:
This one too, I'll check and update other PRs that include these.